types/axe-webdriverjs/index.d.ts

```diff
@@ -83,7 +82,15 @@ export interface AxeBuilder {
     ): Promise<AxeAnalysis>;
 }
 
+export interface BuilderOptions {
+    /**
+     * Should errors be printed into console
+     * @default true
+     */
+    logIframeErrors?: boolean;
+}
+
 export const AxeBuilder: {
```

Comment:
The named export does seem to be wrong if I'm reading the [source](https://unpkg.com/axe-webdriverjs@2.3.0/dist/index.js) correctly... do you know what the deal is with that?

Reply:
I believe it's left here after being added by original author:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26309/files
so because it works, it's untouched.
The original solution was probably to answer the problem:
https://github.com/dequelabs/axe-webdriverjs/issues/74